hack/kind/dns-controller-manager/dns-controller-manager-up.sh

```diff
@@ -20,7 +20,7 @@ download_external_dns_management_helm_charts()
     return
   fi
 
-  wget -qO- https://github.com/gardener/external-dns-management/archive/refs/tags/$VERSION.tar.gz | tar xvz -C ${SOURCE_PATH}/dev external-dns-management-${VERSION#v}/charts/external-dns-management
+  curl -sL "https://github.com/gardener/external-dns-management/archive/refs/tags/$VERSION.tar.gz" | tar xvz -C ${SOURCE_PATH}/dev external-dns-management-${VERSION#v}/charts/external-dns-management
```

Comment:
`-s` = silent/quiet
`-L` = follows redirects

Quoting of the URL to prevent issues with variable expansion or special characters.